### PR TITLE
Release 0.1

### PR DESCRIPTION
**Pull request by palango**
Also updated the release procedure here: https://github.com/raiden-network/team/wiki/%5BWIP%5D-Making-releases#services

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -29,3 +29,20 @@ install:
 
 install-dev:
 	pip install -r requirements-dev.txt
+
+package:
```
Comment (karlb) [inline Makefile]: Is `package` a usual name for this target? Isn't something like `dist` very common?

File: Makefile [hunk 2]
```diff
@@ -29,3 +29,20 @@ install:
...
+
+clean-build:
+	rm -fr build/
+	rm -fr dist/
+	rm -fr .eggs/
+	find . -name '*.egg-info' -exec rm -fr {} +
+	find . -name '*.egg' -exec rm -f {} +
+
+clean-pyc:
+	find . -name '*.pyc' -exec rm -f {} +
```
Comment (karlb) [inline Makefile]:
```suggestion
	find . -name '*.pyc' -delete
```
If that works on OSX, too. That might be a GNU find extension.

File: Makefile (hunk 1 above)
Comment (palango) [reply Makefile]: Sounds good as well

File: Makefile (hunk 2 above)
Comment (palango) [reply Makefile]: Seems to work fine

**Comment by palango** [comment]
> Great! Now, where is our release party?

Most important point! We should at least get some champagne on Thursday!

**Pull request merged**